src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -40,20 +41,30 @@ const InputPassword: UISchemaElementComponent<{
   };
 
   return (
-    <PasswordInput
-      label={t(getLabelName(label!))}
-      value={value}
-      name={name}
-      id={name}
-      error={error !== ''}
-      helperText={error}
-      onChange={handleChange}
-      fullWidth
-      inputProps={{
-        'data-se': name,
-        ...attributes,
-      }}
-    />
+    <Box>
+      <PasswordInput
+        label={t(getLabelName(label!))}
+        value={value}
+        name={name}
+        id={name}
+        error={error !== ''}
+        onChange={handleChange}
+        fullWidth
+        inputProps={{
+          'data-se': name,
+          ...attributes,
+        }}
+      />
+      {error && (
+        <FormHelperText
+          data-se={`${name}-error`}
+          error
+        >
+          {error}
+        </FormHelperText>
+      )}
```

Comment:
The `PasswordInput` should end up handling the error directly. We can remove this additional piece one it's fixed. Add a comment with this ticket number: OKTA-514722

Reply:
tests should be above to catch this issue when mui/odyssey package upgrade, that should be easier to catch than a jira comment